metadata.tvshows.themoviedb.org/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="metadata.tvshows.themoviedb.org"
        name="The Movie Database"
-       version="3.1.3"
+       version="3.1.4"
```

Comment:
The current version is already 3.1.4. Users won't receive updates if you don't change the version number, no?!

Reply:
That's correct, net sure why github shows the old diff :/

Reply:
Maybe because he's missing this commit in his branch: 

https://github.com/xbmc/repo-scrapers/commit/e865afbe3fffb3229a0fa901a6d66d349d470d54#diff-f9f6d4418ec2bc854dbf8a671df80b0d

a rebase might help